src/platform/silabs/wifi/dhcp_client.cpp

```diff
@@ -109,7 +111,7 @@ uint8_t dhcpclient_poll(void * arg)
             dhcp_state = DHCP_ADDRESS_ASSIGNED;
 
             uint64_t addr = netif->ip_addr.u_addr.ip4.addr;
-            ChipLogProgress(DeviceLayer, "DHCP IP: %d.%d.%d.%d", NETIF_IPV4_ADDRESS(addr, 0), NETIF_IPV4_ADDRESS(addr, 1),
+            ChipLogProgress(DeviceLayer, "DHCP IP: %lld.%lld.%lld.%lld", NETIF_IPV4_ADDRESS(addr, 0), NETIF_IPV4_ADDRESS(addr, 1),
```

Comment:
is this print required here? I believe we were printing this in the got_ipv4 function.

Reply:
and the range of IPv4 is 0-255 for each segment which is satisfied with `%d`, not really convinced `%lld` is required here